logging-modules/log4j2/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+    <Properties>
+        <Property name="LOG_PATTERN">
+            %d{yyyy-MM-dd HH:mm:ss.SSS} --- thread_id="%tid" thread_name="%tn" thread_priority="%tp" --- [%p] %m%n
+        </Property>
+    </Properties>
+    <Appenders>
+        <RollingFile name="RollingName" filename="logs/app.log" filePattern="$${date:yyyy-MM}/app-%d{MM-dd-yyyy}-%i.log">
+            <PatternLayout>
+                <Pattern>${LOG_PATTERN}</Pattern>
+            </PatternLayout>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="20MB" />
```

Comment:
Applying the formatter the spaces between value and closing field dissapears:
`<SizeBasedTriggeringPolicy size="20MB"/>`

Reply:
Done